### PR TITLE
Adding checkcred and voidcred actions to VCert CLI

### PR DESCRIPTION
**Pull request by tr1ck3r**
This PR adds the functionality requested by https://github.com/Venafi/vcert/issues/150 and https://github.com/Venafi/vcert/issues/151 via new `checkcred` and `voidcred` actions.

```
# ./vcert getcred -u https://tpp.venafi.example --username local:admin --password somePassw0rd!
vCert: 2021/01/24 23:01:21 Getting credentials...
access_token:  KIKro8hL/4EvvSeVuJQVlw==
access_token_expires:  2021-04-24T23:01:23Z
refresh_token:  PqIQ+4KAEFHGkHZM95b3og==
#
# ./vcert checkcred -u https://tpp.venafi.example -t KIKro8hL/4EvvSeVuJQVlw==
vCert: 2021/01/24 23:01:43 Checking credentials...
access_token_expires:  2021-04-24T23:01:22Z
grant_expires:  2022-01-24T23:01:22Z
client_id:  vcert-cli
scope:  certificate:manage,revoke
#
# ./vcert voidcred -u https://tpp.venafi.example -t KIKro8hL/4EvvSeVuJQVlw==
vCert: 2021/01/24 23:02:02 Access token grant successfully revoked
```

### CODE REVIEW

**Review by angelmoo — APPROVED** [review]
Changes looks good

File: cmd/vcert/commands.go [hunk 1]
```diff
@@ -318,73 +346,127 @@ func doCommandGetcred1(c *cli.Context) error {
...
+				jsonData, err := json.MarshalIndent(resp, "", "    ")
+				if err != nil {
+					return err
+				}
+				fmt.Println(string(jsonData))
+			} else {
+				tm := time.Unix(int64(resp.Expires), 0).UTC().Format(time.RFC3339)
+				fmt.Println("access_token: ", resp.Access_token)
+				fmt.Println("access_token_expires: ", tm)
+				fmt.Println("refresh_token: ", resp.Refresh_token)
```
Comment (marcos-albornoz) [inline cmd/vcert/commands.go:375]: is it not needed to do a nil validation like is in line 396 and 418?

File: cmd/vcert/commands.go [hunk 2]
```diff
@@ -318,73 +346,127 @@ func doCommandGetcred1(c *cli.Context) error {
...
+				}
+				fmt.Println(string(jsonData))
+			} else {
+				tm := time.Unix(int64(resp.Expires), 0).UTC().Format(time.RFC3339)
+				fmt.Println("access_token: ", resp.Access_token)
+				fmt.Println("access_token_expires: ", tm)
+				if resp.Refresh_token != "" {
+					fmt.Println("refresh_token: ", resp.Refresh_token)
+				}
+			}
```
Comment (marcos-albornoz) [inline cmd/vcert/commands.go:399]: can this block of code be converted to a common func? I see that is used in more than one place.

File: cmd/vcert/commands.go (hunk 1 above)
Comment (rvelaVenafi) [reply cmd/vcert/commands.go:375]: isnt line 356 the nil validation?

File: cmd/vcert/commands.go (hunk 2 above)
Comment (tr1ck3r) [reply cmd/vcert/commands.go:399]: The `else` block varies by which action is specified (`getcred`, `checkcred`, and `voidcred`).  The code for the JSON output is common but it is only 5 lines on code (3 for checking error) so I didn't think it was necessary.  Do you think so?

File: cmd/vcert/commands.go (hunk 1 above)
Comment (marcos-albornoz) [reply cmd/vcert/commands.go:375]: that is right :-)

**Review by rvelaVenafi — APPROVED** [review]

File: cmd/vcert/commands.go (hunk 2 above)
Comment (marcos-albornoz) [reply cmd/vcert/commands.go:399]: yes, I though the same, but also I though that if the code for the output changes/increment, then could be more easy to do the change only in one place reducing the risk to forgot to do the change in the places where the code logic is common. that is why I'm asking about if you consider to do that change :-)

**Review by marcos-albornoz — APPROVED** [review]

File: cmd/vcert/commands.go (hunk 2 above)
Comment (tr1ck3r) [reply cmd/vcert/commands.go]: Added `outputJSON` function to address

**Pull request merged**